### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by Durgesh00**
Registration form with test cases

### CODE REVIEW

File: src/components/ButtonWrapper.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from 'reactstrap';
+import PropTypes from 'prop-types';
+const ButtonWrapper = ({ style, onClick, buttonText }) => {
+  return (
+    <Button className={style} onClick={onClick}>
+      {buttonText}
+    </Button>
+  );
+};
```
Comment (sahilbhatia) [inline src/components/ButtonWrapper.js:10]: There is a much simpler way of defining wrapper. To be used untill any specific customizations are required.

```
import { Button } from 'reactstrap';

export default React.memo(Button);
```

File: src/components/FormCard.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    },
+    address: {
+      field: 'exampleAddress',
+      labelText: 'Address',
+      type: 'address',
+      name: 'address',
+      placeholder: ''
+    }
+  };
+  const { firstname, lastname, email, password, country, state, city, address } = FormAttributes;
```
Comment (sahilbhatia) [inline src/components/FormCard.js]: This should be coming through props

File: src/components/FormCard.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      placeholder: ''
+    },
+    address: {
+      field: 'exampleAddress',
+      labelText: 'Address',
+      type: 'address',
+      name: 'address',
+      placeholder: ''
+    }
+  };
```
Comment (sahilbhatia) [inline src/components/FormCard.js]: Should not be defined in this component

File: src/components/FormCard.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            email={email}
+            password={password}
+            country={country}
+            state={state}
+            city={city}
+            address={address}
+          />
+          {/* <h6>Already registered? login</h6> */}
+        </CardBody>
+      </Card>
```
Comment (sahilbhatia) [inline src/components/FormCard.js]: Move this out to it's own component

File: src/components/FormCard.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  } else {
+    return (
+      <Card>
+        <CardBody>
+          <h3>Login</h3>
+          <hr />
+          {/* <LoginForm email={email} password={password}/> */}
+          {/* <h6>Not registered? register</h6> */}
+        </CardBody>
+      </Card>
```
Comment (sahilbhatia) [inline src/components/FormCard.js]: Move this out to it's own component

**Review by sahilbhatia — CHANGES_REQUESTED** [review]
Please check comments and apply to all other components.

File: package.json [hunk 6]
```diff
@@ -38,7 +38,7 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^7.8.1",
+    "eslint": "^6.6.0",
```
Comment (sahilbhatia) [inline package.json:41]: Why did we downgrade the eslint version?

**Review by sahilbhatia — CHANGES_REQUESTED** [review]

File: src/components/FormCard.js (hunk 3 above)
Comment (DevKaranPradhan) [reply src/components/FormCard.js]: moved to their own components

File: src/components/FormCard.js (hunk 4 above)
Comment (DevKaranPradhan) [reply src/components/FormCard.js]: added wrappers of card and cardbody

**Comment by DevKaranPradhan** [comment]
closed

**Pull request closed without merging**